src/StatesDumper/StatesDumper/config.json

```diff
@@ -4,6 +4,7 @@
     "BlockCacheSize": 1000,
     "HeightToBegin": 0,
     "HeightToStartRealTimeSyncing": -1,
+    "Network": 5195086,
```

Comment:
I think we can support multiply networks for this plugin.

Reply:
Done